roles/oem/overlay/provision_dell_raid_overlay/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# We need to run apt-get update because /var/cache/apt/lists gets removed
+- name: Update apt cache
+  command: "apt-get update"
+
+- name: Install required packages
+  apt: name="{{ item.package }}={{ item.version }}"
+  with_items: raid_overlay_package_manifest
+
+# Install perccli tools
+- name: Copy perccli tool
+  copy: src="{{ playbook_dir }}/files/{{ raid_overlay_perccli_package }}" dest=/opt/
+
+- name: Install perccli tools
+  command: dpkg -i /opt/{{ raid_overlay_perccli_package }}
+
+- name: Remove perccli tool
```

Comment:
Maybe call this "Remove perccli package" so it's not confusing that we're removing the installed tool?